index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Hello Microverse</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <nav>
+            <a class="logo" href="#">
+                <img src="images/MyLogo.svg" alt="logo">
+            </a>
+            <div class="onion">
+                <img src="images/onion.svg" alt="">
+            </div>
+        </nav>
+        <section>
+            <h1 class="intro primary-text1">
+                I am Dagmawi, Glad to see you
+            </h1>
```

Comment:
- [ ] The wrong color and font-weight are being used here. Double-check the Figma template for the right ones.